Gemfile

```diff
@@ -23,6 +23,9 @@ gem 'mysql2'
 # for observing records
 gem 'rails-observers'
 
+# for rating
+gem 'ratyrate', github: 'wazery/ratyrate', branch: 'master'
```

Comment:
What's this master branch version you guys are talking about? I'm trying the master, but keep comes the alpha, is there another file to download? 

I'm having a lot of issues with this gem for helper `rating_for_user`. I can't record at `overrall_averages` and `average_cache` tables, as long record the` rater_id` at `rate table. `
The method on this file is also comment:
https://github.com/wazery/ratyrate/blob/master/lib/ratyrate/model.rb

tks